template/{% if AddAllcontributors %}.all-contributorsrc{% endif %}.jinja

```diff
@@ -0,0 +1,5 @@
+{
+"projectName": {{ PackageName }},
+"projectOwner": {{ PackageOwner }},
+"files": ["README.md", "docs/src/index.md"]
+}
```

Comment:
This should be fixed by pre-commit.

Reply:
I did run the pre-commit this time, indentation still seems not to be fixed. I dont know why.